### ISSUE TITLE
Add ":from" parameter in vacation command automatically

### ISSUE THREAD

**Issue by rcubetrac** [roundcube/roundcubemail#5077]
_Reported by Bibby on 3 Jul 2013 00:57 UTC as Trac ticket #1489221_

Plugin: managesieve.

According to RFC document, Sieve Email Filtering: Vacation Extension: https://tools.ietf.org/html/rfc5230#section-5.4:
```
5.4.  From

   Unless explicitly overridden with a :from parameter, the From field
   SHOULD be set to the address of the owner of the Sieve script.
```

So, please consider add a ":from" parameter automatically (value should be current login username), or either let user be able to choose one from drop-down list of current identities.

_Keywords: managesieve, vacation, from_
_Migrated-From: http://trac.roundcube.net/ticket/1489221_

**Comment by iredmail** [comment]
Dear support,

Dovecot will use envelope recipient for `:from`, but could you please help add an input field in `managesieve` plugin to allow user to specify other from address? It's useful if you have some restriction rules in your company, and only some sender addresses are allowed to send/relay email.

We do have request from our client:
http://www.iredmail.org/forum/topic10809-iredmail-support-rewrite-header-for-sieve-forwardersvacation.html

**Comment by alecpl** [comment]
Already implemented in 162bc85be9de.
